Sources/SwifterSwift/Foundation/MeasurementExtensions.swift

```diff
@@ -0,0 +1,52 @@
+// MeasurementExtensions.swift - Copyright 2020 SwifterSwift
```

Comment:
uhm.. technically its's 2021, but not sure if and how important that is .-.

Reply:
Yeah, but it feels special to be the first to use 2021. Bye-bye 2020!